src/main.ts

```diff
@@ -133,22 +136,22 @@ export function wrap<T>(
   }
 
   if (
-    has(cloudFunction, '__trigger.httpsTrigger') &&
-    get(cloudFunction, '__trigger.labels.deployment-callable') !== 'true'
+    !!cloudFunction?.__trigger?.httpsTrigger &&
+    cloudFunction?.__trigger?.labels['deployment-callable'] !== 'true'
```

Comment:
Your index access needs to be null coalescing here.

Reply:
Whups! Thanks for the catch!